doc/content/reference/configuration/application-server.md

```diff
@@ -71,6 +85,15 @@ Application Server can use Javascript payload formatters to decode uplink and en
 ## Application Packages Options
 
 - `as.packages.workers`: Number of workers per application package to process requests (default 1024)
+- `as.packages.timeout`: Message processing timeout (default `10s`)
+
+## Azure IoT Hub Integration Options
+
+{{< distributions "Cloud" "Enterprise" >}}
+
+- `as.packages.azure-iot-hub.cluster.application-server`: Application Server cluster address (default `localhost`)
+- `as.packages.azure-iot-hub.cluster.join-server`: Join Server cluster address (default `localhost`)
+- `as.packages.azure-iot-hub.cluster.network-server`: Network Server cluster address (default `localhost`)
```

Comment:
@adriansmares this is confusing to me; is this correct?

Reply:
Are you referring to the fact that these settings exist ?

The AS uses the `as.packages.azure-iot-hub.cluster.*-server` parameters in order to fill the Application/Join/Network Server address in the Identity Server when a device is registered via an Azure IoT Hub.

This is related probably to our cluster address template discussion.